src/histolab/slide.py

```diff
@@ -580,35 +539,49 @@ def __len__(self) -> int:
 
     # ---public interface methods and properties---
 
-    def save_scaled_slides(self, scale_factor: int = 32, n: int = 0) -> None:
-        """Save rescaled images
+    def scaled_slides(
+        self, scale_factor: int = 32, n: int = 0
+    ) -> List[PIL.Image.Image]:
+        """Return rescaled slides.
```

Comment:
i don't like this name, actually we're returning a list of pil images not a list of slides, isn't it? Maybe a better name for this would be worthwile